cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -100,6 +106,47 @@ def optimization_at(
         )
 
 
+@cirq.transformer
+def strategy_executor(
+    circuit: 'cirq.Circuit',
+    *,
+    execution_strategy: ExecutionStrategy = None,
+    context: Optional['cirq.TransformerContext'] = None,
+) -> None:
+    """Executes an acquaintance strategy"""
+
+    if execution_strategy is None:
+        raise ArgumentError(execution_strategy, 'Execution strategy cannot be None')
```

Comment:
In Cirq, the convention is to use `ValueError` instead of `ArgumentError` to signal that the value of an argument is wrong. Please change to `ValueError`.